dts/bindings/sensor/adi,adltc2990.yaml

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) 2023 Carl Zeiss Meditec AG
+# SPDX-License-Identifier: Apache-2.0
+
+
+description: ADLTC2990 Quad I2C Voltage, Current and Temperature Monitor
+
+compatible: "adi,adltc2990"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+  reg:
+    required: true
+    description: |
+      i2c device address
```

Comment:
This is redundant (already defined in included `i2c-device.yaml`), please remove.

Reply:
done